api/system-settings/execute

```diff
@@ -47,15 +47,6 @@ case $action in
         password=$(echo $data | jq -r '.SmartHostPassword')
         port=$(echo $data | jq -r '.SmartHostPort')
         tls=$(echo $data | jq -r '.SmartHostTlsStatus')
-        [[ $tls = 'true' ]] && tls='--ssl' || tls=''
-        /usr/bin/curl smtp://$hostname:$port -v -k --connect-timeout 10 --max-time 10 --mail-from $username --mail-rcpt $username --user "$username:$password" $tls <<EOF
-Subject: Test smarthost credentials
-Date: $(date -R)
-Message-ID: testEmail.$(date +%s)@$(hostname -d)
-From: no-reply@$(hostname -d)
-To: $username
-
-Test mail for smarthost credentials.
-EOF
-        exit $?
+        [[ $tls = 'true' ]] && tls='--tls' || tls='--tls-optional-strict'
+        /usr/bin/swaks -a -q AUTH -au $username -ap $password --server $hostname --port $port $tls
```

Comment:
Protect shell special chars.

```suggestion
        /usr/bin/swaks -a -q AUTH -au "$username" -ap "$password" --server "$hostname" --port "$port" "$tls"
```

Reply:
Does it work with passwords like

    pass"word
    pass ; rm /word
    pass & word
    ...